### ISSUE TITLE
Cannot load dmd stable dub as project

### ISSUE THREAD

**Issue by rikkimax** [intellij-dlanguage/intellij-dlanguage#709]
**Desktop (please provide environment info):**
 - OS: Windows 10
 - IDE: Intellij 2022.1.2 Community
 - Plugin Version 1.28

**D compiler and tools (please provide version numbers):**
 - dmd: 1.100

**Describe the problem**
Cannot load dmd ~stable dub file as project.

Crashes without any messages.

**Please provide full details to reproduce the problem**

1. Clone dmd, stable branch.
2. Open dub file as project.

**Additional context**
[threadDump-20220613-090751.txt](https://github.com/intellij-dlanguage/intellij-dlanguage/files/8886829/threadDump-20220613-090751.txt)

**Comment by SingingBush** [comment]
this is another issue related to _DubConfigurationParser::getDubProject_ which often takes a long time due to the call to `dub describe`.

https://github.com/intellij-dlanguage/intellij-dlanguage/issues?q=is%3Aissue+is%3Aopen+DubConfigurationParser

I think doing #571 will make a good improvement but need to get some time to try it out. 

```
"AWT-EventQueue-0" prio=0 tid=0x0 nid=0x0 waiting on condition
     java.lang.Thread.State: TIMED_WAITING
 on java.util.concurrent.FutureTask@efd1900
	at java.base@11.0.15/jdk.internal.misc.Unsafe.park(Native Method)
	at java.base@11.0.15/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:234)
	at java.base@11.0.15/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:444)
	at java.base@11.0.15/java.util.concurrent.FutureTask.get(FutureTask.java:203)
	at io.github.intellij.dlanguage.project.DubConfigurationParser.getDubProject(DubConfigurationParser.java:81)
	at io.github.intellij.dlanguage.codeinsight.dcd.DCDCompletionServer.buildDcdCommand(DCDCompletionServer.java:183)
	at io.github.intellij.dlanguage.codeinsight.dcd.DCDCompletionServer.spawnProcess(DCDCompletionServer.java:108)
	at io.github.intellij.dlanguage.codeinsight.dcd.DCDCompletionServer.exec(DCDCompletionServer.java:98)
	at io.github.intellij.dlanguage.codeinsight.dcd.DCDCompletionServer.moduleAdded(DCDCompletionServer.java:257)
	at com.intellij.openapi.module.impl.ModuleImpl.lambda$moduleAdded$2(ModuleImpl.java:257)
```
